docs/help/getting-started/index.md

```diff
@@ -1,56 +1,11 @@
 
-### Step 1: Create your space/documentation
+Our guides below will help you locate and use those files to deploy your API docs on Bump.sh:
 
-After creating your account, you'll be asked [to create and name](https://bump.sh/docs/new?utm_source=bump&utm_medium=content_hub&utm_campaign=getting_started) your first API documentation.
-By default, your documentation URL will use the same name: you can customize it in the slug field. You can also use [custom domains](custom-domains.md), which we'll explain later.
-
-![](/files/help/doc-creation.png)
-
-### Step 2: Upload your file
-
-If you already have a specification file, you can immediately upload it.
-You can also use our examples to see Bump.sh in action. Finally, our [CLI](bump-cli.md) can also be directly used to jump in.
-
-![](/files/help/upload-spec-file.png)
-
-### Step 3: Enjoy
-
-Generating your first documentation will take a few seconds. But let's not stop here: many features are awaiting to customize and make the most of your API documentation. Let's have a look in the next section.
-
-
-## Publish your documentation
-
-Congratulations on releasing your first API documentation on Bump.sh!
-
-One of the first things you may have in mind is how to share your documentation with the world, your team or keep it for yourself.
-Bump.sh offers several options to manage access to your documentation.
-During the onboarding, you may have noticed the option of making your doc public or private.
-
-Public documentation are available to anyone with the URL or if the search engine indexation has been activated.
-Private ones are only accessible to you (or your team/organization if you have one).
-
-We share more details on the [Access Management page](access-management.md).
-
-## Customization options
-
-With your documentation ready, you could want to make it more customized. Bump.sh offers several options to do so.
-
-You can setup [Custom Domains](custom-domains.md), to make your doc available behind your own URL.
-Several branding options are available: from your logo to [Meta Images](meta-images.md) (that helps sharing your doc in a fancy way).
-We also support [Documentation Topics](doc-topics.md), that helps you adding rich (con)text to your API doc, sharing the informations your community needs.
-
-![](/files/help/meta-image-2.png)
-
-## And way more
-
-We do not stop here. Feel free to browse this documentation or directly reach us to find out how to make the most of Bump.sh.
-There's a lot more to cover: from Hubs that groups several API documentation to Branching, hosting several version of a same doc, you'll find many simple options that make a huge change.
-
-The team is always here to listen if you have any feature requests or suggestions. We are here to help.
-
-**We hope you'll enjoy using Bump.sh!**
+- [Quick start with definition files](quick-start) if you work directly with an OpenAPI or AsyncAPI file.
```

Comment:
I know that's the solution we agreed on to... ease the changes for everyone.
However, I'm still sad to add an extra page and click for the users. I strongly believe that's a downgrade.

I won't come back on that topic so far, just want to put this here for the record.